pkg/nuctl/command/delete.go

```diff
@@ -103,7 +105,7 @@ func newDeleteFunctionCommandeer(ctx context.Context, deleteCommandeer *deleteCo
 			return deleteCommandeer.rootCommandeer.platform.DeleteFunction(ctx, deleteFunctionOptions)
 		},
 	}
-
+	cmd.Flags().BoolVar(&commandeer.withAPIGateways, "with-api-gateways", false, `Whether function should be removed with its api gateways (default: false)`)
```

Comment:
Why backticks and not quotes?

Reply:
@TomerShor I copied the code from the function below, so that's why but I deleted it already